client/onboarding-prototype/index.tsx

```diff
@@ -37,12 +42,36 @@ const Step: React.FC< Props > = ( { name, children } ) => {
 const OnboardingStepper = () => {
 	const { data } = useOnboardingContext();
 
-	const handleComplete = () => {
-		const { connectUrl } = wcpaySettings;
-		const url = addQueryArgs( connectUrl, {
-			progressive: fromDotNotation( data ),
+	const isEligibleForPo = async () => {
+		// TODO GH-5475 maybe move somewhere, not sure if it's the best place for this
+		const businessDetails: ProgressiveOnboardingEligibleFields = {
```

Comment:
Is this approach decent (to prepare the request data)?
Took example from https://github.com/Automattic/woocommerce-payments/blob/3781aaf06f6d2519bf069c6de3d6aee62d3e984d/client/vat/form/tasks/company-data-task.tsx#L63-L63

Reply:
Looks good to me, although I'll write it like:
```ts
export interface EligibleData {
	business: {
		country: string;
		type: string;
		mcc: string;
		annual_revenue: string;
		go_live_timeframe: string;
	};
}
```

As we're using it within PO files, I think we can get rid of `ProgressiveOnboarding` from names, otherwise we will end with pretty long ones 😄  To ensure that it meets the data structure, `business` should be the parent. And as all of those properties are required, you should remove the optional definition (`?`)

Reply:
Addressed in https://github.com/Automattic/woocommerce-payments/commit/81a0430e86b35d74a140e4916e0368747bfff8b3